DOC-6397-Sync-Gateway-missing stats

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-6397

- Amended per feedback
- added-rev-cache-to-index
- Also add section headers into page
- Added another-tweak-to-links

ibsoln committed:
DOC-6397-Sync-Gateway-missing stats

https://issues.couchbase.com/browse/DOC-6397

DOC-6397 Amended per feedback

https://issues.couchbase.com/browse/DOC-6397

DOC-6397-added-rev-cahe-to-index
Also add section headers into page

DOC-6397-and-another-tweak-to-links

ibsoln committed:
DOC-6397-address-review comments
- attachment_pull_bytes
- num_pull_repl_since_zero also needs updating to new description

ibsoln reviewed (commented):
Per Jacques https://couchbase.slack.com/archives/D010WQHS4UC/p1586193100003600

ibsoln commented:
Ready to merge need an and-1 approver

simon-dew commented on modules/ROOT/pages/stats-monitoring.adoc:
```diff
@@ -1,18 +1,32 @@
+
+You can request the monitoring stats, from the Admin REST API, using the `expvars` method, as shown in this `curl` command:
 
 [source,java]
```
Should be [source,console]

ibsoln replied:
Amended

simon-dew commented on modules/ROOT/pages/stats-monitoring.adoc:
```diff
@@ -117,113 +183,130 @@ The response is a JSON object with the following schema.
+The CPU's utilization as percentage value.
+Derivation::
+The CPU usage calculation is performed based on _user_ and _system_ CPU time, but it doesn't include components such as `iowait`.
+This means that the values of `process_cpu_percent_utilization` and `%Cpu`, returned when running the `top` command, will *differ*  .
```
Delete space after `*differ*` and before full stop.

ibsoln replied:
Change

simon-dew commented on modules/ROOT/pages/stats-monitoring.adoc:
```diff
@@ -117,113 +183,130 @@ The response is a JSON object with the following schema.
+
 === per_db
 
 Stats for each database declared in the config file
```
Make use of full stops consistent throughout.

simon-dew commented on modules/ROOT/pages/stats-monitoring.adoc:
```diff
@@ -232,291 +315,657 @@ Stats for each database declared in the config file
+==== chan_cache_hits
+Description::
+Channel cache requests fully served by the cache.
+Use Case::
```
Make capitalisation consistent throughout. (`Use case` vs. `Use Case`)

ibsoln replied:
Changed to Use Case throughout

simon-dew reviewed (changes requested):
Just minor changes: markup, consistency of punctuation and capitalisation

ibsoln commented:
Cheers, I’ll look at addressing those thing today


From: Simon Dew <notifications@github.com>
Reply-To: couchbase/docs-sync-gateway <reply@reply.github.com>
Date: Tuesday, 7 April 2020 at 17:23
To: couchbase/docs-sync-gateway <docs-sync-gateway@noreply.github.com>
Cc: "Ian Bridge [C]‌" <ian.bridge@couchbase.com>, Author <author@noreply.github.com>
Subject: Re: [couchbase/docs-sync-gateway] DOC-6397-Sync-Gateway-missing stats (#210)


@simon-dew requested changes on this pull request.

Just minor changes: markup, consistency of punctuation and capitalisation

________________________________

In modules/ROOT/pages/stats-monitoring.adoc<https://github.com/couchbase/docs-sync-gateway/pull/210#discussion_r404933838>:

>

-The following `curl` command requests the monitoring stats on the Admin REST API.

+* Performance

+* Resource utilization and

+* Health

+

+You can request the monitoring stats, from the Admin REST API, using the `expvars` method, as shown in this `curl` command:



 [source,java]

Should be [source,console]

________________________________

In modules/ROOT/pages/stats-monitoring.adoc<https://github.com/couchbase/docs-sync-gateway/pull/210#discussion_r404936102>:

>

-CPU utilization (%).

+Description::

+The CPU's utilization as percentage value.

+Derivation::

+The CPU usage calculation is performed based on _user_ and _system_ CPU time, but it doesn't include components such as `iowait`.

+This means that the values of `process_cpu_percent_utilization` and `%Cpu`, returned when running the `top` command, will *differ*  .

Delete space after *differ* and before full stop.

________________________________

In modules/ROOT/pages/stats-monitoring.adoc<https://github.com/couchbase/docs-sync-gateway/pull/210#discussion_r404936667>:

>  Number of warnings logged.



+== Metrics by Database

+

+{xref-cb-config}cache[cache] | {xref-cb-config}cbl_replication_pull[cbl_replication_pull] | {xref-cb-config}cbl_replication_push[cbl_replication_push] | {xref-cb-config}database[database] | {xref-cb-config}delta_sync[delta_sync] | {xref-cb-config}gsi_views[gsi_views] | {xref-cb-config}security[security] | {xref-cb-config}shared_bucket_import[shared_bucket_import]

+

 === per_db



 Stats for each database declared in the config file

Make use of full stops consistent throughout.

________________________________

In modules/ROOT/pages/stats-monitoring.adoc<https://github.com/couchbase/docs-sync-gateway/pull/210#discussion_r404937716>:

>

-===== chan_cache_misses

+==== chan_cache_hits

+Description::

+Channel cache requests fully served by the cache.

+Use Case::

Make capitalisation consistent throughout. (Use case vs. Use Case)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/couchbase/docs-sync-gateway/pull/210#pullrequestreview-389273695>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMSVPQXDWAPXLOMZMDLTMKTRLNHP7ANCNFSM4L3ZHZLA>.

ibsoln committed:
DOC-6397 address review comments

https://issues.couchbase.com/browse/DOC-6397

https://github.com/couchbase/docs-sync-gateway/pull/210#pullrequestreview-389273695

Led to many cosmetic changes as I rewrote sections to make a tapestry of changes by mutiple authors
into  a homgenous document

ibsoln committed:
DOC-6397-Sync-Gateway-missing-stats

https://issues.couchbase.com/browse/DOC-6397

Add alphabetic indexing

ibsoln committed:
DOC-6397-Sync-Gateway-missing-stats

https://issues.couchbase.com/browse/DOC-6397

Chabges arising from pre merge checks
